docs/UserGuide.md

```diff
@@ -416,42 +529,48 @@ distinctions, even if they share an email address.
 
 **Q**: Why is it possible to add applications with dates that have already passed?<br>
 **A**: Users are permitted to add applications with past dates because we're in the process of developing new 
-[features](#future-advancements). These features will enable users to take notes about specific companies, which they can
+[features](#future-enhancements-hammer). These features will enable users to take notes about specific companies, which they can
 refer back to when considering future applications.
 
 **Q**: When is a company classified as a duplicate?<br>
 **A**: A company is identified as a duplicate when you attempt to add a record with identical name, phone number, email 
-address, tags, start date, and end date as an existing entry. If all these fields match, the system flags it as a duplicate,
+address, tags, start date, and end date as an existing entry. If all these fields matches, the system flags it as a duplicate,
 preventing the addition of such an application.
 
+**Q**: Why do applications with no dates appear between sorted applications by start or end date?
+**A**: We apologise for the confusion caused. This occurs because some applications may have only a start date or an end 
+date entered, causing them not to display any dates. However, when sorting by start date, applications with only a start 
+date will also be considered in the sorting process.
+
 --------------------------------------------------------------------------------------------------------------------
 
-## Known issues
+## Known issues :construction:
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
 2. **Only when both start and end dates are present, will the date be shown in the GUI**. If only one of the dates is present, the sorts will take them into account while sorting, leading to misleading errors such as the one below.
 
-<div style="text-align:center;">
-    <img src="images/datesorterror.png" alt="datesorterror" width="400" height="200">
-</div>
+![date sort](images/datesorterror.png)
 
 3. **Only when both start and end dates are present, will the date be shown in the GUI**, this allows users to seemingly add in duplicate companies when in fact the start/end dates are different.
 4. **Duplicate tags**, are allowed to be added into the same company, however they will still only show unique tags. 
```

Comment:
for this maybe we can show an image to make it easier for the user to understand what is duplicate tags in the command